packages/@n8n/api-types/src/chat-hub.ts

```diff
@@ -290,3 +290,25 @@ export interface EnrichedStructuredChunk extends StructuredChunk {
 		executionId: number | null;
 	};
 }
+
+const chatProviderSettingsSchema = z.object({
+	provider: chatHubProviderSchema.exclude(['n8n', 'custom-agent']),
+	enabled: z.boolean().optional(),
+	credentialId: z.string().nullable(),
+	limitModels: z.boolean(),
```

Comment:
Do you think this can be redundant, since limitModels=true with empty allowedModels doesn't make sense (but UI can still have the toggle)